user_sync/rules.py

```diff
@@ -617,8 +679,16 @@ def write_remove_list(self, file_path, dashboard_users):
                 writer.writerow({'user': username, 'domain': domain})
                 total_users += 1
         self.logger.info('Total users in remove list: %d', total_users)
-                
-        
+
+    def log_after_mapping_hook_scope(self, before_call):
+        when = 'before' if before_call else 'after'
+        if (before_call):
+            self.logger.debug('.')
```

Comment:
what is this about?  are you trying to get a separator line in the log?  I bet we could be a lot clearer than a simple period.